.github/workflows/ci.yml

```diff
@@ -38,4 +38,4 @@ jobs:
       - name: Sync
         run: uv sync --all-extras
       - name: Test
-        run: uv run pytest tests
+        run: uv run pytest -k test_pdc
```

Comment:
I think this should be switched back to all tests

Reply:
@emmanuelmathot This will be removed during merge. It was done only to test CI for particular source.